tests/test_audit_analyser.py

```diff
@@ -11,15 +12,30 @@
 
 
 @patch.object(Config, "get_s3_audit_report_key", return_value="s3")
+@patch.object(Config, "get_github_audit_report_key", return_value="github_admin_report")
 class TestAuditAnalyser(TestCase):
-    def test_check_s3_compliance(self, _: Mock) -> None:
+    def test_check_s3_compliance(self, *_: Mock) -> None:
         notifications = {notification(item="item-1"), notification(item="item-2")}
         audit = Audit(type="s3", report=[{"report": "val-1"}, {"report": "val-2"}])
 
         with patch.object(S3Compliance, "analyse", return_value=notifications) as s3_compliance:
             self.assertEqual(notifications, AuditAnalyser().analyse(audit, Config()))
         s3_compliance.assert_called_once_with(audit)
 
-    def test_check_unsupported_audit(self, _: Mock) -> None:
+    def test_check_github_compliance(self, *_: Mock) -> None:
+        notifications = {notification(item="item-1"), notification(item="item-2")}
+        audit = Audit(
+            type="github_admin_report",
+            report=[
+                {"name": "some-repo", "branchProtectionRules": []},
+                {"name": "some-repo-2", "branchProtectionRules": []},
+            ],
+        )
```

Comment:
This can be simplified as this doesn't really have to be representative of an actual report (the `analyse` method is mocked):

```suggestion
        audit = Audit(type="github_admin_report", report=[{"name": "some-repo"}, {"name": "some-repo-2"}])
```